src/tzdb.rs

```diff
@@ -557,14 +558,14 @@ impl TzProvider for FsTzdbProvider {
             LocalTimeRecordResult::Empty => Vec::default(),
             LocalTimeRecordResult::Single(r) => {
                 let epoch_ns =
-                    EpochNanoseconds::try_from(epoch_nanos.0 + seconds_to_nanoseconds(r.offset))?;
+                    EpochNanoseconds::try_from(epoch_nanos.0 - seconds_to_nanoseconds(r.offset))?;
```

Comment:
Huh, is this correct? I think the tests below due assert that the time zones are not inverted like in PosixTzString, so I'm pretty sure the sign needs to be minus (I could definitely be off here). Ex: EST is -04:00, to move the local back to UTC you local - -04:00.

Reply:
Yes. The input is a local time, and the output is an offset from UT. Hence, since `local = UT + offset`, then `local - offset = UT`

Reply:
OH wait, I had put plus oh god :rofl: